docs/docsite/rst/dev_guide/developing_modules_general_windows.rst

```diff
@@ -388,6 +389,126 @@ at the end of the file. For example
     Export-ModuleMember -Function Invoke-CustomUtil, Get-CustomInfo
 
 
+Exposing shared module options
+++++++++++++++++++++++++++++++
+
+PowerShell module utils can easily expose common module options that a module can use when building its argument spec.
+This allows common features to be stored and maintained in one location and have those features used by multiple
+modules with minimal effort. Any new features or bugifxes added to one of these utils are then automatically used by
+the various modules that call that util.
+
+An example of this would be to have a module util that handles authentication and communication against an API This
+util can be used by multiple modules to expose a common set of module options like the API endpoint, username,
+password, timeout, cert validation, etc, without having to add those options to each module spec.
+
+The standard convention for a module util that has a shared argument spec would have it
+
+- Add a ``Get-<namespace.name.util name>Spec`` function that outputs the common spec for a module
+    * It is highly recommended to make this function name be unique to the module to avoid any conflicts with other utils that can be loaded
+    * The format of the output spec is a Hashtable in the same format as the ``$spec`` used for normal modules
+- A function that takes in an ``AnsibleModule`` object called under the ``-Module`` parameter which it can use to get the shared options
```

Comment:
> The standard convention for a module util that has a shared argument spec **would have it**

Was this sentence cut off prematurely?

Or maybe just remove `it` and then remove `Add` from the first bullet point so they match?

Reply:
Thanks for the pickup, I just removed the `it` and `Add` as you suggested.